tools/tplg_parser/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(sof_source_directory "${PROJECT_SOURCE_DIR}/../..")
 add_library(sof_tplg_parser SHARED tplg_parser.c)
 target_include_directories(sof_tplg_parser PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/include)
 target_include_directories(sof_tplg_parser PRIVATE ${sof_source_directory}/src/include)
-target_compile_options(sof_tplg_parser PRIVATE -g -O -Wall -Werror -Wl,-EL -Wmissing-prototypes -Wimplicit-fallthrough=3)
+target_compile_options(sof_tplg_parser PRIVATE -g -O -Wall -Werror -Wl,-EL -Wmissing-prototypes -Wimplicit-fallthrough)
```

Comment:
Off-topic but since commit f7f9d5f42e58 we should gradually stop hardcoding -g and -O flags and let CMAKE_BUILD_TYPE manage these.

Reply:
Agreed, let's move that to a bug or dev discussion

Reply:
#3467